code/backend/batch/utilities/helpers/EnvHelper.py

```diff
@@ -51,20 +51,32 @@ def __init__(self, **kwargs) -> None:
         # Azure OpenAI
         self.AZURE_OPENAI_RESOURCE = os.getenv("AZURE_OPENAI_RESOURCE", "")
         self.AZURE_OPENAI_MODEL = os.getenv("AZURE_OPENAI_MODEL", "")
-        self.AZURE_OPENAI_MODEL_NAME = os.getenv("AZURE_OPENAI_MODEL_NAME", "")
-        self.AZURE_OPENAI_TEMPERATURE = os.getenv("AZURE_OPENAI_TEMPERATURE", "")
-        self.AZURE_OPENAI_TOP_P = os.getenv("AZURE_OPENAI_TOP_P", "")
-        self.AZURE_OPENAI_MAX_TOKENS = os.getenv("AZURE_OPENAI_MAX_TOKENS", "")
+        self.AZURE_OPENAI_MODEL_NAME = os.getenv(
+            "AZURE_OPENAI_MODEL_NAME", "gpt-35-turbo"
+        )
+        self.AZURE_OPENAI_TEMPERATURE = os.getenv("AZURE_OPENAI_TEMPERATURE", 0)
```

Comment:
Same with these three

Reply:
Again, these two were defaulted to those values in the app.py. Happy to make them strings if you think they should be.